pkg/ui/static/js/graph.js

```diff
@@ -32,11 +32,18 @@ Prometheus.Graph.stepValues = [
   "1w", "2w", "4w", "8w", "1y", "2y"
 ];
 
-Prometheus.Graph.numGraphs = 0;
+// Get id of last panel and increment.
+let num = 0;
+Object.entries(localStorage).map(([key]) => {
+  if (key.includes("enable-dedup") && num < parseInt(key[key.length - 1], 10)) {
+    num = parseInt(key[key.length - 1], 10);
+  }
```

Comment:
```suggestion
  if (key.includes("enable-dedup") && num < Number(key[key.length - 1])) {
    num = Number(key[key.length - 1]);
  }
```

parseInt is usually not preferred as it can cause problems in some cases.